cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,283 @@
+import sys
+import json
+import argparse
+from os import linesep as LINESEP
+
+# backward compatibility with 2.x
+try:
+    ModuleNotFoundError
+except:
+    ModuleNotFoundError = ImportError
+
+try:
+    import rados
+except ModuleNotFoundError as noModErr:
+    print("Error: %s\nExiting the script..." % noModErr)
+    sys.exit(1)
+
+try:
+    # for 2.7.x
+    from StringIO import StringIO
+except ModuleNotFoundError:
+    # for 3.x
+    from io import StringIO
+
+
+class ExecutionFailureException(Exception):
+    pass
+
+
+class RadosJSON:
+    EXTERNAL_USER_NAME = "client.healthchecker"
+
+    @classmethod
+    def gen_arg_parser(cls):
+        argP = argparse.ArgumentParser()
+        argP.add_argument("--conf", "-c",
+                          help="Provide a ceph conf file.", type=str)
+        argP.add_argument("--run-as-user", "-u",
+                          help="Provides a user name to check the cluster's health status",
+                          default=cls.EXTERNAL_USER_NAME, type=str)
+        argP.add_argument("--json", "-j", action="store_true",
+                          default=False, help="Provides a JSON output")
+        argP.add_argument("--bash", "-b", action="store_true",
+                          default=False, help="Provides BASH script compatible output")
+        argP.add_argument("--output", "-o", default="",
+                          help="Output will be stored into the provided file")
+        argP.add_argument("--cephfs-filesystem-name", default="", required=True,
+                          help="Provides the name the filesystem")
+        argP.add_argument("--cephfs-data-pool-name", default="", required=True,
+                          help="Provides the name of the cephfs data pool")
+        argP.add_argument("--rdb-datapool-name", default="", required=True,
+                          help="Provides the name of the RDB datapool")
+        argP.add_argument("--namespace", default="",
+                          required=True, help="Namespace where CephCluster is running")
+        argP.add_argument("--rgw-pool-prefix", default="",
+                          help="RGW Pool prefix")
+        argP.add_argument("--rgw-realm-name", default="",
+                          help="RGW Realm name")
+        argP.add_argument("--rgw-zonegroup-name", default="",
+                          help="RGW ZoneGroup name")
+        argP.add_argument("--rgw-zone-name", default="", help="RGW Zone name")
+        argP.add_argument("--rbd-data-pool-name", default="",
+                          help="RBD data pool name")
+        argP.add_argument("--rgw-endpoint", default="", required=True,
+                          help="Rados GateWay endpoint")
```

Comment:
The should reflect that we need an IP:PORT value.

Reply:
Added that part in the `help` string